.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+# For issues that have been open for awhile without activity, label
+# them as stale with a warning that they will be closed out. If
+# anyone comments to keep the issue open, it will automatically
+# remove the stale label and keep it open.
+
+# Runs once a day.
+
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    uses: fivetran/dbt_package_automations/.github/workflows/stale-bot.yml@22a7ff2 #update to @main once it's merged in the central repo
```

Comment:
In the past we referenced a branch as opposed to a specific commit sha. Is there a reason we changed that up here?

Reply:
I suppose this is moot since I'm going to remove this file, but it's only because it's not merged to main and I felt nervous about the forward slashes in my test branch name, which can be tested on the test package then.